developer_setup/debugging.md

```diff
@@ -0,0 +1,64 @@
+## Debugging ManageIQ with Remote Pry
+
+Interactive debugging using pry or byebug is normally difficult with ManageIQ, as `rake evm:start` launches the server and worker processes in the background. Therefore the only way to do interactive debugging is with `pry-remote` or `byebug` running in remote mode.
+
+Running ManageIQ with `byebug` was exceedingly slow, so this guide will focus on remote debugging using `pry`.
+
+### Adding dependencies
+
+You'll need to add the following gems to your `Gemfile`:
```

Comment:
Instead of requesting the user to modify the Gemfile, prefer creating/modifying `Gemfile.dev.rb` or create a file in the `bundler.d` directory